_raml/apps/products/examples/get_product_productid_crossselling_response.json

```diff
@@ -369,6 +370,7 @@
         "refUnit": "piece",
         "refContentAmount": 1,
         "priceQuantity": 1,
+        "isInWishlist": false,
```

Comment:
The placement of the attribute is a bit odd. I would rather place it between `searchKeywords` and `links`. What do you think?